src/modules/Channel/components/MessageList/index.tsx

```diff
@@ -120,29 +120,22 @@ const MessageList: React.FC<MessageListProps> = ({
   };
 
   /**
-   * Move the messsage list scroll
-   * when each message's height is changed by `reactions` OR `showEdit`
+   * 1. Move the messsage list scroll
+   *    when each message's height is changed by `reactions` OR `showEdit`
+   * 2. Keep the scrollBottom value after fetching new message list
    */
-  const handleMessageHeightChange = () => {
+  const moveScroll = (_isBottomMessageAffected?: boolean): void => {
+    const isBottomMessageAffected = _isBottomMessageAffected ?? false;
```

Comment:
```suggestion
  const moveScroll = (isBottomMessageAffected = false): void => {
```
This should be fine too :)

Reply:
aha! thank you~